prime-router/metadata/hl7_mapping/common/observation-result.yml

```diff
@@ -212,7 +212,7 @@ elements:
     resourceIndex: contactIndex
 
   - name: observation-note
-    condition: '%observation.note.exists()'
+    condition: '%context.note.exists()'
```

Comment:
Changed this one spot so that there was a code example that could be referenced.